### ISSUE TITLE
"Backup keys" should be "Export keys"

### ISSUE THREAD

**Issue by notramo** [syphon-org/syphon#622]
Settings → Security & Privacy → Backup keys

It should be called "Export keys". "Backup" terminology is used by Element for server-side backup, which is not supported in Syphon. This can cause confusion, potentially leaving users with missing keys (they think it's backed up on server, but not).

**Comment by dnisbetjones** [comment]
Excellent point and can make the already confusing key situation more confusing for newer users.

**Comment by ereio** [comment]
While Element does consider "Backup" to mean server-side backup, I think it should be explicit you're saving your encryption keys somewhere else outside the device, even if they are encrypted before being uploaded. I intended on having a separate section to enable "Cloud Backups" that would explicitly mention saving them to the home server.

Happy to chat about this though, and even revert back to "Export" for the time being until server-side backups are implemented

**Comment by notramo** [comment]
I agree with "Cloud Backup" to indicate it's saved to server, however, local backup should be called "Export" instead of "Backup" to indicate that keys are saved outside of app, into a file.

**Comment by ereio** [comment]
For tracking purposes:

We had a convo in the dev chat about this and came to an agreement on:

- Local Backup
- Local Backup Schedule (for auto backups)
- Cloud Backup

I'll switch the current copy to "Export" for now until we can support both.
